src/cityreader/cityreader.py

```diff
@@ -61,11 +73,23 @@ def cityreader(cities=[]):
 # TODO Get latitude and longitude values from the user
 
 def cityreader_stretch(lat1, lon1, lat2, lon2, cities=[]):
```

Comment:
The tests for the stretch passed! Great work, Nikki!
<img width="1661" alt="Screen Shot 2019-12-09 at 2 08 57 AM" src="https://user-images.githubusercontent.com/16110931/70414739-f0409600-1a28-11ea-9cfe-ae7dac19a063.png">